src/main/java/com/develhope/spring/vehicle/optionals/models/AllowWheels.java

```diff
@@ -0,0 +1,37 @@
+package com.develhope.spring.vehicle.optionals.models;
+
+import jakarta.persistence.*;
+
+@Entity
+@Table(name = "Wheels")
+public class AllowWheels {
```

Comment:
Ok guardando meglio ho scoperto che il nome corretto della classe dovrebbe essere Rim che vuol dire cerchioni. Mentre alloyWheels vuol dire proprio cerchi in lega